src/gluonts/dataset/repository/_tsf_datasets.py

```diff
@@ -238,7 +238,7 @@ def default_prediction_length_from_frequency(freq: str) -> int:
         "T": 60,
         "H": 48,
         "D": 30,
-        "W": 8,
+        "W-SUN": 8,
```

Comment:
This fix doesn't specific to `dominick`, but be an issue with the canonical name of the weekly frequency.

Otherwise looks good to me.

Reply:
Yes, I was trying to find out which other dataset it affects, but there aren't many dataset. I think it comes from a specific condition arising out of the Dominick dataset. I marked it as Dominick, because yesterday, I encountered issue because of it.

Reply:
Should `W-SUN` be added along with `W`, instead of replacing?

Reply:
Yes. I think that would be better. :)

Reply:
Updated.